core/controllers/custom_landing_pages.py

```diff
@@ -38,6 +38,7 @@ def get(self, subject, topic):
         """Handles GET requests."""
         if subject in feconf.AVAILABLE_LANDING_PAGES:
             if topic in feconf.AVAILABLE_LANDING_PAGES[subject]:
+                self.values['title'] = topic.title()
```

Comment:
Let's go with capitalize(). I spent a fair amount of time trying to understand why title() was a function in the first place :)

Reply:
Removed this line of code. (Done!)